tests/test_emulators.py

```diff
@@ -92,18 +100,59 @@ def test_simple(self):
 
         model(TS)
 
+        # ------------
+        # Check quality (diagnostic) of simulated streamflow values
+        # ------------
         d = model.diagnostics
-        # yields NSE=0.???? for full period 1954-2010
-        assert model.rvi.calendar == "GREGORIAN"
-        # Check parser
-        assert 1 in model.solution["HRUStateVariableTable"]["data"]
-
-        np.testing.assert_almost_equal(d["DIAG_NASH_SUTCLIFFE"], -0.117301, 2)
+        np.testing.assert_almost_equal(d["DIAG_NASH_SUTCLIFFE"], -0.116971, 4)
 
+        # ------------
+        # Check simulated streamflow values q_sim
+        # ------------
         hds = model.q_sim
+
         assert hds.attrs["long_name"] == "Simulated outflows"
 
+        assert len(hds.nbasins == 2)  # ??? WHY IS THAT 2??
+
+        # We only have one SB with gauged=True, so the output has a single column.
+        # The number of time steps simulated between (2000, 1, 1) and
+        # (2002, 1, 1) is 732.
+        assert hds.shape == (732, 1)
+
+        # Check simulated streamflow at first three timesteps and three simulated
+        # timesteps in the middle of the simulation period.
+        dates = (
+            "2000-01-01",
+            "2000-01-02",
+            "2000-01-03",
+            "2001-01-30",
+            "2001-01-31",
+            "2001-02-01",
+        )
+
+        target_q_sim = [0.0, 0.165788, 0.559366, 12.388717, 12.347956, 12.307299]
+
+        print("hds.sel(nbasins=0) = ", hds.sel(nbasins=0))
+
+        for t in range(6):
+            np.testing.assert_almost_equal(
+                hds.sel(nbasins=0, time=dates[t]), target_q_sim[t], 4
+            )
+
+        # ------------
+        # Check parser
+        # ------------
+        assert model.rvi.calendar == "GREGORIAN"
+
+        # ------------
+        # Check saved HRU states saved in RVC
+        # ------------
+        assert 1 in model.solution["HRUStateVariableTable"]["data"]
+
+        # ------------
```

Comment:
Just added the same checks as we do also with `test_routing` for some consistency